nntrainer/meson.build

```diff
@@ -11,6 +11,17 @@ nntrainer_inc_abs = [
   meson.source_root() / 'api' / 'ccapi' / 'include'
 ]
 
+# The code below feeds BiQGEMM absolute path to android.mk
+# The BiQGEMM absolute path is included only when biqgemm is enabled.
+if get_option('enable-biqgemm')
```

Comment:
Just to be sure, this won't affect `tools/package_android.sh`, isn't it?

Reply:
Yes. This won't be affect to the current `tools/package_android.sh`. This aims for the case where `meson build -Dplatform=android -Denable-biqgemm=true`.